File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist1.cs
// <snippetdistributecampaignfrommarketinglist1>


 // Copy the campaign.
 var campaignCopyRequest = new CopyCampaignRequest
 {
     BaseCampaign = _originalCampaignId
 };

 var copyCampaignResponse =
     (CopyCampaignResponse)_serviceProxy.Execute(campaignCopyRequest);
 _campaignId = copyCampaignResponse.CampaignCopyId;

 Console.WriteLine("  Copied the campaign to new campaign with GUID \r\n\t{{{0}}}",
     _campaignId);

// </snippetdistributecampaignfrommarketinglist1>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist1.vb
' <snippetdistributecampaignfrommarketinglist1>


	' Copy the campaign.
          Dim campaignCopyRequest = New CopyCampaignRequest With
                                    {
                                        .BaseCampaign = _originalCampaignId
                                    }

          Dim copyCampaignResponse =
              CType(_serviceProxy.Execute(campaignCopyRequest), 
                  CopyCampaignResponse)
	_campaignId = copyCampaignResponse.CampaignCopyId

          Console.WriteLine("  Copied the campaign to new campaign with GUID " _
                            &amp; vbCrLf &amp; vbTab &amp; "{{{0}}}", _campaignId)

' </snippetdistributecampaignfrommarketinglist1>